scripts/test_import_migration.py

```diff
@@ -0,0 +1,63 @@
+# Tester for the flaskext_migrate.py module located in flask/scripts/
+#
+# Author: Keyan Pishdadian
+import pytest
+from redbaron import RedBaron
+import flaskext_migrate as migrate
+
+
+def test_simple_from_import():
+    red = RedBaron("from flask.ext import foo")
+    output = migrate.fix_tester(red)
+    assert output == "import flask_foo as foo"
+
+
+def test_from_to_from_import():
+    red = RedBaron("from flask.ext.foo import bar")
+    output = migrate.fix_tester(red)
+    assert output == "from flask_foo import bar as bar"
+
+
+def test_multiple_import():
+    red = RedBaron("from flask.ext.foo import bar, foobar, something")
+    output = migrate.fix_tester(red)
+    assert output == "from flask_foo import bar, foobar, something"
+
+
+def test_multiline_import():
+    red = RedBaron("from flask.ext.foo import \
+                    bar,\
+                    foobar,\
+                    something")
+    output = migrate.fix_tester(red)
+    assert output == "from flask_foo import bar, foobar, something"
+
+
+def test_module_import():
+    red = RedBaron("import flask.ext.foo")
+    output = migrate.fix_tester(red)
+    assert output == "import flask_foo as foo"
+
+
+def test_named_module_import():
+    red = RedBaron("import flask.ext.foo as foobar")
+    output = migrate.fix_tester(red)
+    assert output == "import flask_foo as foobar"
+
+
+def test__named_from_import():
+    red = RedBaron("from flask.ext.foo import bar as baz")
+    output = migrate.fix_tester(red)
+    assert output == "from flask_foo import bar as baz"
+
+
+def test_parens_import():
+    red = RedBaron("from flask.ext.foo import (bar, foo, foobar)")
+    output = migrate.fix_tester(red)
+    assert output == "from flask_foo import (bar, foo, foobar)"
+
+
+def test_function_call_migration():
+    red = RedBaron("flask.ext.foo(var)")
+    output = migrate.fix_tester(red)
+    assert output == "flask_foo(var)"
```

Comment:
Okay, there are a few problems with this. Let me work on it. Thanks for looking it over.